contributing/development/compiling/compiling_for_linuxbsd.rst

```diff
@@ -363,6 +363,71 @@ You don't even need to copy them, you can just reference the resulting
 files in the ``bin/`` directory of your Godot source folder, so the next
 time you build, you automatically have the custom templates referenced.
 
+Compiling for RISC-V
+--------------------
+
+To compile Godot for RISC-V, we need to setup the following items:
```

Comment:
This needs to be clarified if this documentation is for cross-compiling, native compiling with an older toolchain, or both. If this includes native compiling and something is not necessary for native compiling, that part needs to be noted (for example, I compiled natively without any extra toolchains and that worked fine for me).

Reply:
These instructions are for *cross-compiling* (as I've noted in the description of this PR). However, I also assumed that someone may try this strategy even if they're on a native device.

But I'll clarify that the focus is cross-compilation.